app/test_split_integer.py

```diff
@@ -2,20 +2,33 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(32, 6)) == 32
```

Comment:
There are some problems with grouping tests like that:
- Assertions chained inside one function won't execute if the previous one fails. You generally should use `pytest.mark.parametrize()` if you're using pytest.
- Adding more assertions for the same functionality is fine, but not strictly necessary. If the cases you test for are not executed in the function in different ways, you testing for just one of them is often considered enough.